vuefrontend/src/views/Home.vue

```diff
@@ -202,6 +207,7 @@ export default {
           this.sameStations = true;
           this.missingStations = false;
           this.validSearch = false;
+          this.fetchingResult = false;
```

Comment:
This will let the user request multiple times if if stmt is true.
This should be removed or if stmt should return